### PR TITLE
In-spect scanner tweaks

### PR DESCRIPTION
**Pull request by cacogen**
## About The Pull Request

I saw one on the floor today and the description was unfortunate.

**Does the following:**

- Spellchecks and simplifies the description a bit. Hopefully it reads as well as it can
- Renames it the N-spect scanner, which seemed to me to be a better version of the type of thing the person who added it was going for
- Changes the sound effect from the quaint voiced placeholder-seeming one to something in the same vein I found on Freesound https://freesound.org/people/Tissman/sounds/522328/
- Changes the report to be a child of paper
- Adds encrypted text to the paper

## Why It's Good For The Game

Makes it better

## Changelog
:cl: cacogen
tweak: In-spect scanner is now called N-spect scanner (the N stands for Nanotrasen)
soundadd: N-spect scanner has a better sound effect
spellcheck: N-spect scanner description reads better
code: N-spect scanner report is now a child of paper
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -5. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by cacogen** [comment]
@ArcaneMusic 

I wasn't aware this was yours until after I made this just now. What do you think?

**Comment by ArcaneMusic** [comment]
Seems fine far as I can tell.
Sound replacement is probably for the best (The soul has been CRUSHED underFOOT and is BLEEDING OUT PROFUSELY)
I originally considered making the output paper but I decided against it since it would probably need like an internal paper level system or something for reloading, but that's niche enough and not upheld on anything else, so we can certainly live without it.
Name change and description is good, flavor is upheld and seems need.

God fucking damnit it's just WGW.

File: code/modules/cargo/bounties/security.dm [hunk 1]
```diff
@@ -45,7 +45,7 @@
 	name = "Routine Security Inspection"
 	description = "Perform a routine security inspection using an in-spect scanner on the following general area on station:"
```
Comment (cacogen) [inline code/modules/cargo/bounties/security.dm:46]:
```suggestion
	description = "Perform a routine security inspection using an N-spect scanner on the following station area:"
```

**Comment by CameronWoof** [comment]
"Central Command-issued" would probably read more clearly.

**Comment by cacogen** [comment]
Yeah I wasn't sure whether to put a hyphen there or not but I think you're right.

File: code/game/objects/items/documents.dm [hunk 2]
```diff
@@ -58,8 +58,9 @@
 			update_icon()
 
 /obj/item/inspector
-	name = "in-spect scanner"
-	desc = "Cental commmand issued inspection device. Does company grade station inspection protocols when activated, and prints encripted sheets of paper regarding the mainenance of the station. Hard to Replace."
+	name = "\improper N-spect scanner"
+	desc = "Central Command issued inspection device. Performs inspections according to Nanotrasen protocols when activated, then \
```
Comment (cacogen) [inline code/game/objects/items/documents.dm:62]:
```suggestion
	desc = "Central Command-issued inspection device. Performs inspections according to Nanotrasen protocols when activated, then \
```

File: code/game/objects/items/documents.dm [hunk 3]
```diff
@@ -77,23 +78,40 @@
...
+
+	info = random_string(rand(180,220), characters)
+	info += "[prob(50) ? "=" : "=="]" //Based64 encoding
 
-/obj/item/report/examine(mob/user)
+/obj/item/paper/report/examine(mob/user)
 	. = ..()
 	if(scanned_area?.name)
-		. += "<span class='notice'>\The [src] contains data on [scanned_area.name].</span>"
+		. += "<span class='notice'>\The [src] contains data on \the [scanned_area.name].</span>"
```
Comment (cacogen) [inline code/game/objects/items/documents.dm:110]:
```suggestion
		. += "<span class='notice'>\The [src] contains data on [scanned_area.name].</span>"
```

File: code/game/objects/items/documents.dm [hunk 4]
```diff
@@ -77,23 +78,40 @@
...
-	desc = "Contains detailed information about the station's current status, too bad you can't really read it."
+	desc = "Contains no information about the station's current status."
 	icon = 'icons/obj/bureaucracy.dmi'
-	icon_state = "slipfull"
+	icon_state = "slip"
 	///What area the inspector scanned when the report was made. Used to verify the security bounty.
 	var/area/scanned_area
+	show_written_words = FALSE
+
+/obj/item/paper/report/proc/generate_report(var/area/scan_area)
```
Comment (cacogen) [inline code/game/objects/items/documents.dm:94]:
```suggestion
/obj/item/paper/report/proc/generate_report(area/scan_area)
```

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-3). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Comment by interestingusernam3** [comment]
woody

**Pull request merged**